topics/transcriptomics/tutorials/mirna-target-finder/slides.html

```diff
@@ -0,0 +1,166 @@
+- This figure is an illustration of the quasi-mapping of a read using k=3. Hash table lookup of k-mer A T T returns the suffix array interval b and e.
+- The current k-mer and its matching suffix intervals are colored in green.
+- The base G at position 6 has a mismatch with the C on reference transcripts represents a possible sequencing error.
+- Hence, the first five bases A T T G A is the maximal mappable prefix and the suffix array interval of this maximal mappable prefix is b' and e'.
+- A T T G A C T A which is the red colored box is the longest common prefix of the suffix array interval b' and e'. So the next informative position on the read is the 9th base T.
+- In the end, the read in the above example most likely map to the suffix array at e'.
+
+---
+
+### Salmon advantages and limitations
+
+- Adapter clipping can be skipped
+- Consider trimming low-quality reads
+- Fast and accurate quantification
+- Suitable for well-annotated organisms
+- Cannot be used to detect or annotate novel isoforms/transcripts
+
+???
+- Salmon has several advantages and few limitations over the traditional alignment methods.
+- Since the k-mers which contain the adaptor sequences are not present in the transcriptome index from which the hash table is generated, they cannot be mapped.
+- Hence, if there is no significant amount of adapters present in the reads, adapter clipping can safely be skipped.
+- However, it is always good to trim the low-quality reads.
+- Salmon provides fast and accurate quantification.
+- Because of the large memory footprints of hash tables and suffix arrays, generally, transcriptome FASTA file is used for Salmon quantification instead of whole genome FASTA file.
+- Therefore it is advisable to use this tool on well-annotated organisms.
+- For the same reason, Salmon cannot be used to detect or annotate novel transcripts.
+
+---
+
+### plant small RNA target prediction using TargetFinder
+
+- Inputs
+    - Small RNA sequence
+    - FASTA-formated target sequence database file
+- Uses Smith-Waterman algorithm to align query small RNA sequence to the target sequence database
+- Penalty of 1 for mismatches, single-nucleotide gaps, or single-nucleotide bulges
+- Penalty of 0.5 G U base pairs
+- Doubled penalty scores at positions 2-13 relative to the 5' end of the small RNA query sequence
+- Duplexes with the following characteristics are rejected:
+    - more than one single-nucleotide bulge or gap
+    - more than seven total mismatches, G:U base pairs, bulges and gaps
+    - more than four total mismatches or four total G:U base pairs
+
+???
+- In animals usually, base pairing in the seed region of the micro RNA, that is 2 to 8 bases is enough for target recognition.
+- Plant micro RNAs require more stringent base pairing. Generally, a near-perfect pairing in 5 prime region and a substantial pairing in 3 prime region is necessary.
```

Comment:
```suggestion
- Plant miRNAs require more stringent base pairing. Generally, a near-perfect pairing in 5 prime region and a substantial pairing in 3 prime region is necessary.
```

Reply:
could just be added to the ari-map.yml

Reply:
do I have to add definitions for both the `miRNA` and `MiRNA` or is it case insensitive?

Reply:
It should be insensitive